testsuite/python/scafacos_dipoles_1d_2d.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (C) 2010,2011,2012,2013,2014,2015,2016 The ESPResSo project
+
+#  
+# This file is part of ESPResSo.
+#  
+# ESPResSo is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#  
+# ESPResSo is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#  
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.   
+
+
+# This tests the scafacos p2nfft dipolar calculations by matching against
+# reference data from direct summation. In 2d, reference data from the mdlc
+# test case is used
+
+
+
+from __future__ import print_function
+import espressomd 
+import espressomd.magnetostatics as magnetostatics
+import numpy as np
+import unittest as ut
+
+
+class Scafacos1d2d(ut.TestCase):
+    if "SCAFACOS_DIPOLES" in espressomd.features():    
```

Comment:
Please use the `skipIf` decorator as in
````python
@ut.skipIf(not espressomd.has_features(["SCAFACOS_DIPOLES"]),
           "Features not available, skipping test!")
class Scafacos1d2d(ut.TestCase):
````
Saves you also one level of indentation inside the class.